src/bot.ts

```diff
@@ -142,6 +152,10 @@ export class TreasureMapBot {
             (hero) => hero.state === "Work" && hero.energy > 0
         );
     }
+    get notWorkingSelection() {
+        console.log(this.squad.notWorking);
```

Comment:
Remove this line please.

Reply:
removido

Reply:
Is it more efficient in getting more bCoin?